lib/model/yake_keywords.py

```diff
@@ -58,6 +61,10 @@ def run_yake(self, text: str,
             language = cld3.get_language(text).language
         ### normalize special characters
         text = self.normalize_special_characters(text)
+        # Segmentation for mandarin
+        if language == 'zh-CN' or language == 'zh' or language == 'zh-TW':
+            text = self.run_chinese_segmentation_with_jieba(text)
+            # text = " ".join(list(jieba.cut_for_search(text)))
```

Comment:
Per above, I would drop the function call and just use `text = " ".join(list(jieba.cut_for_search(text)))` here. If we do stick with the function call, then please remove the commented code.

Reply:
added my comment about this above. please, Let me know what do you think about it.

Reply:
Sounds good. Let's leave as is.